geomstats/learning/riemannian_mean_shift.py

```diff
@@ -1,147 +1,161 @@
-"""Riemannian mean-shift clustering
-
-Lead author: Nina Miolane"""
+"""
+Riemannian mean-shift clustering
+Lead author: Nina Miolane and Shubham Talbar
+"""
 
 import joblib
-import numpy as np
 from sklearn.base import BaseEstimator, ClusterMixin
 
+import geomstats.backend as gs
 from geomstats.learning.frechet_mean import FrechetMean
 
 
 class RiemannianMeanShift(ClusterMixin, BaseEstimator):
-    def __init__(self, manifold, metric, bandwidth, tol=1e-2, **FrechetMean_kwargs):
-        """Class for Riemannian Mean Shift algorithm on manifolds.
-        Mean Shift is a procedure for locating the maxima - the modes of a
-        density function given discrete data sampled from that function. It is
-        an iterative method for finding the centers of a collection of clusters.
-
-        Following implementation assumes a numpy backend and a flat kernel method.
-
-        Parameters
-        ----------
-        manifold : object of class RiemannianManifold
-            The Geomstats Riemannian manifold on which the
-            Riemannian Mean Shift algorithm is to applied.
-        metric : object of class RiemannianMetric
-            The Geomstats Riemannian metric associated to the space used.
-        bandwidth : size of neighbourhood around each center
-            All points in 'bandwidth' size around center are considered for
-            calculating new mean centers.
-        tot : float, stopping condition
-            Computation of subsequent mean centers is stopped when the distance
-            between them is less than 'tot'
-        mean : mean method used to calculate the new center of a cluster
-        """
+    """
```

Comment:
Title needs to be on the first line, right after the quotes """, and there needs to be a blank line between the title and the description.

Reply:
Noted, updated docstring.